akka-docs/src/main/paradox/typed/cluster-sharding.md

```diff
@@ -301,34 +301,55 @@ directly to the `ActorRef`, including messages that the actor sends to itself, a
 
 @@@
 
-Supported passivation strategies are:
+### Idle entity passivation
 
-### Idle passivation strategy
-
-The **idle** passivation strategy passivates entities when they have not received a message for a specified length of
-time. This is the default strategy and is enabled automatically with a timeout of 2 minutes. Specify a different idle
-timeout with configuration:
+Idle entities can be automatically passivated when they have not received a message for a specified length of time.
+This is currently the default strategy, for compatibility, and is enabled automatically with a timeout of 2 minutes.
+Specify a different idle timeout with configuration:
 
 @@snip [passivation idle timeout](/akka-cluster-sharding/src/test/scala/akka/cluster/sharding/ClusterShardingSettingsSpec.scala) { #passivation-idle-timeout type=conf }
 
-Or specify the idle timeout as a duration using the `withIdlePassivationStrategy` method on `ClusterShardingSettings`.
+Or specify the idle timeout as a duration using the `withPassivationStrategy` method on `ClusterShardingSettings`.
+
+Idle entity timeouts can be enabled and configured for any passivation strategy.
+
+### Active entity limits
+
+Automatic passivation strategies can limit the number of active entities. Limit-based passivation strategies use a
+replacement policy to determine which active entities should be passivated when the active entity limit is exceeded.
+
+A recommended passivation strategy, which will become the new default passivation strategy in future versions of Akka
```

Comment:
should we also mention this in the top section `## Automatic Passivation`, where we now only mention "passivate idle entities"?

Reply:
Good idea, will add.

Reply:
Added something in https://github.com/akka/akka/pull/30968/commits/2b60876e46c52c7f37c776759ca28dffcd723c51